src/data/members.json

```diff
@@ -0,0 +1,16 @@
+{
+  "location_1": [
```

Comment:
The data model should look like:
```js
"default_membership": {
  "ID": "default",
  "destinationID" : ""
  "members": [
    "personID": "",
    "personID": ""
       ....
  ]
}
```
Where `ID` is the ID of the membership, `destinationID` is the ID of either the meeting or room, and then the `members` is an array of `personID`s